File: EaseMall-biz/src/main/java/com/ysmull/easemall/exception/PicNotFoundException.java
package com.ysmull.easemall.exception;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

/**
 * @author maoyusu
 */
@ResponseStatus(HttpStatus.NOT_FOUND)
public class PicNotFoundException extends Exception {

    public PicNotFoundException(String message) {
        super(message);
    }


    public PicNotFoundException(String message, Exception cause) {
        super(message, cause);
    }

}

File: EaseMall-biz/src/main/java/com/ysmull/easemall/exception/PicNotFoundException.kt
package com.ysmull.easemall.exception

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus

/**
 * @author maoyusu
 */
@ResponseStatus(HttpStatus.NOT_FOUND)
class PicNotFoundException : Exception {

    constructor(message: String) : super(message) {}


    constructor(message: String, cause: Exception) : super(message, cause) {}

}
